docs/stencil-docs/localization/localizing-stores.md

```diff
@@ -42,7 +42,7 @@ The bulk of localizing a store is translating content on your theme's page templ
 
 ### Multiple language strings by BigCommerce
 
-Some of the strings displayed on your storefront are not currently editable in BigCommerce. These are strings such as "This promotion cannot be applied with the selected currency." BigCommerce will automatically render such strings in the shopper's language as defined by the shopper's browser language settings.
+Some of the strings displayed on your storefront are not currently editable in BigCommerce. These are strings such as "This promotion cannot be applied with the selected currency."  The language setting for these strings are not defined by the shopper's browser language settings. Rather, you should select the language displayed on your storefront by using the default language setting found in the control panel's [Store Profile](https://support.bigcommerce.com/s/article/Store-Profile-Settings#locale) area. This language setting determines the language of your storefront content.

```

Comment:
```suggestion
Strings such as "This promotion cannot be applied with the selected currency"  are not currently editable in BigCommerce. The language settings for these strings are not defined by the shopper's browser language settings. Rather, you select your storefront's display language by using the default language setting found in the control panel's [Store Profile](https://support.bigcommerce.com/s/article/Store-Profile-Settings#locale). This language setting determines the language of your storefront content.

```

I made some changes to make the text more concise but feel free to ignore it if it changes the meaning.